intera_tools_description/urdf/electric_gripper/fingers/basic_soft_tip.xacro

```diff
@@ -20,8 +20,8 @@
       </collision>
       <inertial>
         <origin rpy="0 0 ${pi*(g_reflect+1)/2}" xyz="0 0 0"/>
-        <mass value="2e-06"/>
-        <inertia ixx="1.1e-09" ixy="0" ixz="0" iyy="1.1e-09" iyz="0" izz="1.1e-09"/>
+        <mass value="0.01"/>
```

Comment:
Have you identified what the actual *minimal* requirement for gazebo is in terms of **which links** need to have slightly-more-than-basically-zero mass? If not, could you?

In terms of - is it just the tip links? is it just tip links that are moving? is it just tip links used in chains? is it every link in a chain that is used, or moves? do links after fixed joints not need it?

I know it may be asking for a bit more tedious work, but considering the days spent on this issue, time and time again, it would be great to have the answers to this research written very clearly in the description of this PR.

Reply:
- [ ] For example - the `${side}_gripper_tip` link in the last file doesn't have an updated mass. What are the constraints of the requirements that mean that's okay and these ones need the just-very-small mass?

(/rethink_electric_gripper.xacro)

Reply:
The (mass * inertia tensor) needs to not be too small to zero-out the value when considering numerical precision. Links:
http://gazebosim.org/tutorials?tut=inertia
https://en.wikipedia.org/wiki/List_of_moments_of_inertia#List_of_3D_inertia_tensors